tests/test_renderers.py

```diff
@@ -676,7 +676,7 @@ def get(self, request):
         request = factory.get('/')
         response = view(request)
         response.render()
-        self.assertInHTML('<tr><th>Foo</th><td>a string</td></tr>', str(response.content))
+        self.assertContains(response, '<tr><th>Foo</th><td>a string</td></tr>', html=True)
```

Comment:
how about using pytest assert style?

Reply:
I'm happy to take a look at this suggested change, but I'm not very familiar with all the features of pytest. Do you have a link to documentation on this feature or a short example?